src/Apps/CollectorProfile/Routes/Saves2/Components/__tests__/ArtworkListItem.jest.tsx

```diff
@@ -83,7 +83,7 @@ describe("ArtworkListItem", () => {
     })
 
     const link = screen.getByRole("link")
-    const href = "/collector-profile/saves2/collection-id"
```

Comment:
**suggestion**: let's use BASE_PATH from the [constants file](https://github.com/artsy/force/pull/12255/files#diff-3fae5dc8a2308d9e22e6eed4a0a3763e9e63165f6f46f32054ecd2c0384d6ddd) here?